components/SkillCard.styles.js

```diff
@@ -18,7 +18,7 @@ const styles = css`
 
   .body {
     background-color: #efefef;
-    padding: 0.8rem 0.8rem 2.4rem 2.4rem;
```

Comment:
I don't think it was intentional to have the top and right be `0.8rem` and the bottom and left be `2.4rem`, but let me know if it was.